src/main/java/io/github/bookcrawler/controllers/HelloController.java

```diff
@@ -1,16 +1,31 @@
 package io.github.bookcrawler.controllers;
 
+import ja.crawler.BookExtractor;
+import ja.crawler.Entities.BookInfo;
+import ja.crawler.EmpikBookExtractor;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
 
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.LinkedList;
+import java.util.List;
+
 @Controller
 public class HelloController {
 
     @GetMapping("/")
-    public String hello(Model model) {
+    public String hello(Model model) throws IOException {
+        List<BookInfo> list = new ArrayList<>();
+        BookExtractor empikBookExtractor = new EmpikBookExtractor();
+        Iterable<BookInfo> bookInfos = empikBookExtractor.extract();
+        bookInfos.forEach(bookInfo -> list.add(bookInfo));
+        String title = list.get(0).getTitle();
+        model.addAttribute("title",title);
 
-        model.addAttribute("name", "Result of researching books");
+        //model.addAttribute("name", "Result of researching books");
```

Comment:
Is this comment necessary? I can see your are adding the attribute above, so maybe delete the commented line?